modules/konovalov_i_euler_f_lab2/src/konovalov_i_euler_f.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Konovalov Igor
+
+#include "include/konovalov_i_euler_f.h"
+#include <math.h>
+#include <stdexcept>
+
+bool EulerFunction::contains_divider(int n, std::vector<int>& v) {
```

Comment:
```suggestion
bool EulerFunction::contains_divider(int n, const std::vector<int>& v) {
```
Также добавьте пожалуйста `const` в объявлении